site/styles/_buttons.scss

```diff
@@ -62,9 +62,9 @@
       fill: none;
     }
     path {
-        fill: $button-text-color;
-        stroke: $button-text-color;
-      }
+      fill: $var-text; // Light Theme
```

Comment:
I dislike the `// Light Theme` peppered through all the files - what is the reasoning? If it's to explain that it's part of the theme, I think the renaming of the var solves that.